js/controllers/tab/settings/archiving/form/ArchivingSettingsFormHandler.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @defgroup js_controllers_tab_settings_archiving_form
+ */
+/**
+ * @file js/controllers/tab/settings/archiving/form/ArchivingSettingsFormHandler.js
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2000-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class ArchivingSettingsFormHandler
+ * @ingroup js_controllers_tab_settings_archiving_form
+ *
+ * @brief Handle the press archiving settings form.
+ */
+(function($) {
+
+	/** @type {Object} */
+	$.pkp.controllers.tab.settings.archiving =
+			$.pkp.controllers.tab.settings.archiving || { form: {} };
+
+
+
+	/**
+	 * @constructor
+	 *
+	 * @extends $.pkp.controllers.form.AjaxFormHandler
+	 *
+	 * @param {jQueryObject} $form the wrapped HTML form element.
+	 * @param {Object} options form options.
+	 */
+	$.pkp.controllers.tab.settings.archiving.form.
```

Comment:
I think this will probably result in some JS linting errors due to the indentation being different than JSLint expects. (JSLint is pretty pedantic about this.) Make sure you send in a PR to the application repo that refers to this branch, per the contributing documentation in the wiki, so that the tests run on this and identify anything needing correction.

Reply:
Added the Submodule update ##defstat/archiving-page-changes## commit, waiting for the tests to end.

Reply:
Fixed.